### PR TITLE
Applied Facebook URL Parsing/Reading

### PR DESCRIPTION
**Pull request by hyunsikjeong**

### CODE REVIEW

File: logic/index.ts [hunk 1]
```diff
@@ -8,7 +8,8 @@ import { PlatformAddress } from "codechain-sdk/lib/key/classes";
 export async function giveCCC(
     context: Context,
     to: string,
-    amount: string
+    amount: string,
+    contentId: string
```
Comment (majecty) [inline logic/index.ts]: How about postId?
We are already using "post" to point both tweet and Facebook post.

**Review by majecty — CHANGES_REQUESTED** [review]

File: logic/index.ts (hunk 1 above)
Comment (hyunsikjeong) [reply logic/index.ts]: It's more intuitive than `contentId`. Fixed.

File: routes/api.ts [hunk 2]
```diff
@@ -21,7 +21,23 @@ export function createRouter(context: Context) {
...
+            let content = "";
+
+            if (postType === URLType.Twitter) {
+                content = await getTwitContent(context, postId);
+            }
+            else if (postType === URLType.Facebook) {
+                content = await getFacebookContent(postId);
+            }
+            else {
+                throw new FaucetError(ErrorCode.InvalidURL, null);
```
Comment (majecty) [inline routes/api.ts]: Let's change the error to ErrorCode.Unknown.
If this code is executed, there is a bug, not a user's fault.

**Comment by majecty** [comment]
I get "InvalidURL" error when I write "https://www.facebook.com/majecty/posts/1970196229713167" as an input.
Can you check why the error occurred?

**Review by majecty — CHANGES_REQUESTED** [review]

File: routes/api.ts (hunk 2 above)
Comment (hyunsikjeong) [reply routes/api.ts]: But, for example, some errors can caused by putting an URL with typos. At that time, it is a user's fault, not a bug.
Also, in the case of Twitter, the logical flow of the code is same as the original code. [Link]( codechain-faucet/logic/twitter.ts)

Comment (hyunsikjeong) [reply routes/api.ts]: No, it's my fault. It is a bug. `parseURL` throws an error when the given URL is invalid.

**Comment by hyunsikjeong** [comment]
@majecty Fixed all!

**Pull request merged**